angular/spec.ts

```diff
@@ -0,0 +1,88 @@
+import { browser, element, by } from 'protractor';
+
+let ConvertYM = require('./spec_convert.ts');
```

Comment:
das sind nicht die korrekten Imports, das sollte auch dazu führen, dass deine Dateien falsch geladen werden und dann noch in Typescript referenziert werden.
Import Statements verwenden